docs/wiki/The library - Module design.md

```diff
@@ -463,7 +465,7 @@ Within a bicep file, use the following conventions:
 
 ## Modules
 
-- Module symbolic names are in camel_Snake_Case, following the schema `<mainResourceType>_<referencedResourceType>` e.g., `storageAccount_fileServices`, `virtualMachine_nic`, `resourceGroup_roleAssignments`.
+- Module symbolic names are in camel*Snake_Case, following the schema `<mainResourceType>*<referencedResourceType>`e.g.,`storageAccount_fileServices`, `virtualMachine_nic`, `resourceGroup_roleAssignments`.
```

Comment:
I'm not sure I fully grasp the expression here. Examples make it clear, but what does the * stand for? Shouldn't the schema be `<mainResourceType>_<referencedResourceType>`?

Reply:
This was some damn extension. And I still did not find out which. This must be undone